homeassistant/components/wink/climate.py

```diff
@@ -62,9 +63,17 @@
 SUPPORT_FAN_THERMOSTAT = [FAN_AUTO, FAN_ON]
 SUPPORT_PRESET_THERMOSTAT = [PRESET_AWAY, PRESET_ECO]
 
+HA_HVAC_TO_WINK_AC = {
+    HVAC_MODE_AUTO: "auto_eco",
```

Comment:
This is not a fix. This means that setting HVAC mode to AUTO will always set it to `auto_eco`. The current behavior that htis PR replaces is correct

Reply:
Agreed, I had the wrong understanding of how this state should be represented in HA. I have reworked the pull request to properly handle this as a preset.